### DISCUSSION TITLE
go.sum

### DISCUSSION THREAD

**Discussion by LimbongRobin** [stretchr/testify#1640] [Q&A]
## Description

how to solve this issue
C:\Program Files\Go\pkg\mod\github.com\stretchr\testify@v1.9.0\assert\assertions.go:20:2: missing go.sum entry for module providing package github.com/davecgh/go-spew/spew (imported by github.com/stretchr/testify/assert); to add:
        go get github.com/stretchr/testify/assert@v1.9.0

**Comment by HaraldNordgren** [comment]
Try running ‘go mod tidy’
